html/index.html

```diff
@@ -0,0 +1,480 @@
+                        </p>
+    
+                    </div>
+    
+    
+                    <div class="canopy-list">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+    
+                </div>
+    
+                <div >
+                    <button id="second-container-button">See Project</button>
+                </div>
+    
+            </div>
+    
+    
+    
+        </div>
+    
+        <div class="second-containerl">
+    
+            <div class="inner-second-containerl">
+    
+                <div class="top-image">
+    
+                    <img src="../assets/Snapshoot Portfolio 4.png" alt="">
+    
+                </div>
+    
+                <div class="text-inner-container">
+    
+                    <h2> Multi-Post <br> Stories </h2>
+    
+                    <div class="canopy-inner-container">
+    
+                        <p id="text1">CANOPY</p>
+    
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+    
+                    </div>
+    
+                    <div class="canopy-daily">
+    
+                        <p>
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required.
+        
+                        </p>
+    
+                    </div>
+    
+    
+                    <div class="canopy-list">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+    
+                </div>
+    
+                <div >
+                    <button id="second-container-button">See Project</button>
+                </div>
+    
+            </div>
+    
+        </div>
+    
+        <div class="third-container">
+    
+            <div class="lower-body">
+    
+                <h2>About <br> Myself</h2>
+        
+                <p>
+                  I’m a software developer! I can help you build a product , feature or
+                  website Look through some of my work and experience! If you like what
+                  you see and have a project you need coded, don’t hestiate to contact
+                  me.
+                </p>
+        
+              </div>
+        
+              <div class="footer-third-container">
+                
+                <p><a href="#">LET'S CONNECT</a> </p>
+        
+              </div>
+        
+              <div class="lower-footer">
+        
+                <img src="../assets/twitter.svg" alt="">
+                <img src="../assets/linkdn.svg" alt="">
+                <img src="../assets/medium.svg" alt="">
+                <img src="../assets/github.svg" alt="">
+                <img src="../assets/rabbit.svg" alt="">
+        
+              </div>
```

Comment:
- [ ] At this point, the list of icons must use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag), please fix this on the go.